### ISSUE TITLE
blank white video but audio works

### ISSUE THREAD

**Issue by bugrasan** [codebicycle/videospeed#190]
- OS: windows 11
- Firefox: v100.0.1
- videospeed: v0.6.3.3

when playinging a video on docs.microsoft.com [1] with enabled videospeed, then the video is blank white, however the audio works fine.
if the host `www.microsoft.com` is added to disabled sites list, the video works fine again.

[1] e.g. https://docs.microsoft.com/en-us/learn/modules/sustainable-software-engineering-overview/7-energy-proportionality

it seems that the 'vsc-controller' overlay is causing the problem. when deleting it, the video is visible again.

**Comment by gediminasel** [comment]
Yep, it uses `div:first-child` attribute to style the video stream, but vsc-controller becomes the first child when added. Not sure what a good fix for this could be.
Same issue with the Chrome version [0.7.2](https://github.com/igrigorik/videospeed/commit/b12e1a722e2b641a6cb9fbe3f859d8ccd936e3ea) and [0.8.2-beta](https://github.com/ChadBailey/videospeed-refactoring). You should report it there if it's not already reported.

**Comment by JoshuaIyalagha** [comment]
Similar issue. If you "hide", the screen should show again
![Screen Shot 2022-11-01 at 9 01 40 PM](https://user-images.githubusercontent.com/28325837/199370925-bbbc66d9-ff4a-4fa1-a327-cc14292a4adb.png)

**Comment by xnileshx** [comment]
Same issue with https://learn.microsoft.com 
Has there been any update on this? 
Looks like the only workaround is as @JoshuaIyalagha mentioned to enable " Hide controller by default  " under extension settings.
